### DISCUSSION TITLE
can you take out specific features and make them bookmarks?

### DISCUSSION THREAD

**Discussion by OzzyCantu** [ScratchAddons/ScratchAddons#4088] [Help] [answered]
So recently I asked a question to similar regards on the scratch discussion form but to my dismay my topic was deleted and I got a message from the scratch team. but here is my question

So I am using a school issued chromebook and scratch addons is a blocked extension so I was wondering if it was possible if you could take a certain part of the github, like the onion skinning feature and put it into a bookmark so when you you press it you can use the feature. I have seen this done with online games like google snake where they mod it and when you press the bookmark it doesn't open a new tab but instead it runs a script and mods the game.

Soooo is this possible?

**Comment by cobaltt7** [comment, marked as answer]
unfortunately most features (especially editor ones) would not work in this form. we may make SA into a userscript for use with tampermonkey (https://chrome.google.com/webstore/detail/dhdgffkkebhmkfjojejmpbldmpobfkfo) if that's not blocked. in the mean time, why not try turbowarp (https://turbowarp.org)?

**Reply by OzzyCantu** [reply]
Well funny that you mention turbo wrap, haha. The reason I actually asked this question because turbowarp was blocked :( but if you find any alternatives for turbowarp please do so inform me :)

**Reply by CST1229** [reply]
Try [mirror.turbowarp.xyz](https://mirror.turbowarp.xyz).

**Reply by retronbv** [reply]
normal turbowarp is banned for my school…. staging.turbowarp.org or mirror.turbowarp.xyz works (at least for me)

**Reply by OzzyCantu** [reply]
> Try [mirror.turbowarp.xyz](https://mirror.turbowarp.xyz).

Oooh Thanks that works!
